meta-refkit-computervision/recipes-computervision/librealsense/files/0001-Add-missing-includes-for-functional.patch

```diff
@@ -0,0 +1,42 @@
+From 7b1581644c5fbdec3a22cd512557b391dc74ebf0 Mon Sep 17 00:00:00 2001
+From: Till Hofmann <thofmann@fedoraproject.org>
+Date: Wed, 22 Mar 2017 14:23:24 +0100
+Subject: [PATCH] Add missing includes for <functional>
+
+Newer GCC versions (>=7) fail with an error if the header is not
+included an std::function is used.
+
+Upstream-status: Backport
```

Comment:
Mind the spelling (Upstream-Status) and that "Backport" usually has more information about what was backported. For example:

    libxml2-fix_NULL_pointer_derefs.patch:Upstream-Status: Backport [https://git.gnome.org/browse/libxml2/commit/?id=e905f08123e4a6e7731549e6f09dadff4cab65bd]